src/ucp/rndv/rndv_rtr.c

```diff
@@ -86,7 +86,10 @@ static size_t ucp_proto_rndv_rtr_pack(void *dest, void *arg)
     rtr->address = (uintptr_t)req->send.state.dt_iter.type.contig.buffer;
 
     rpriv = req->send.proto_config->priv;
-    ucs_assert(rpriv->md_map == req->send.state.dt_iter.type.contig.reg.md_map);
+    if (rtr->size > 0) {
+        ucs_assert(rpriv->md_map ==
+                   req->send.state.dt_iter.type.contig.reg.md_map);
+    }
```

Comment:
when can it be 0? If it is for 0 byte msg, can't we always reply ATS?

Reply:
fixed and added min length for rtr